integration-tests/contracts/ethereum_vrfv2_contracts.go

```diff
@@ -445,34 +521,20 @@ func (v *EthereumVRFCoordinatorV2) PendingRequestsExist(ctx context.Context, sub
 }
 
 func (v *EthereumVRFCoordinatorV2) OracleWithdraw(recipient common.Address, amount *big.Int) error {
-	opts, err := v.client.TransactionOpts(v.client.GetDefaultWallet())
-	if err != nil {
-		return err
-	}
-	tx, err := v.coordinator.OracleWithdraw(opts, recipient, amount)
-	if err != nil {
-		return err
-	}
-	return v.client.ProcessTransaction(tx)
+	_, err := v.client.Decode(v.coordinator.OracleWithdraw(v.client.NewTXOpts(), recipient, amount))
+	return err
 }
 
 // OwnerCancelSubscription cancels subscription,
 // return funds to the subscription owner,
 // down not check if pending requests for a sub exist,
 // outstanding requests may fail onchain
 func (v *EthereumVRFCoordinatorV2) OwnerCancelSubscription(subID uint64) (*types.Transaction, error) {
-	opts, err := v.client.TransactionOpts(v.client.GetDefaultWallet())
-	if err != nil {
-		return nil, err
-	}
-	tx, err := v.coordinator.OwnerCancelSubscription(
-		opts,
+	// Do not wrap in Decode() to avoid waiting until the transaction is mined
+	return v.coordinator.OwnerCancelSubscription(
```

Comment:
why you don't want to wait?

Reply:
@Tofel it's because I want to finish this function asap and call https://github.com/smartcontractkit/chainlink/blob/TT-1057-Migrate-VRF-smoke-tests/integration-tests/smoke/vrfv2_test.go#L512

Reply:
@Tofel noe that the old code did not wait here either

Reply:
@lukaszcl old code never waited for anything anywhere, because it was asynchronous.

Reply:
imho we shouldn't do that. All transactions should be synchronous, no exceptions. `WaitForSubscriptionCanceledEvent` this is only needed, because old client was async. You could make cancelling synchronous (with `Decode()`) and just remove that `WaitForSubscriptionCanceledEvent` call.